firebase-appdistribution/src/main/java/com/google/firebase/appdistribution/UpdateProgress.java

```diff
@@ -28,21 +28,23 @@
   }
 
   /**
-   * The number of bytes downloaded so far for the APK. Returns -1 if called on an AAB or if no new
-   * release is available.
+   * The number of bytes downloaded so far for an APK.
+   *
+   * @returns -1 if called when updating to an AAB or if no new release is available.
```

Comment:
Can we include the normal case in this `@returns` statement? Something like "@returns the number of bytes, or -1 if..."